connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/outbound/OutboundConnectorRuntimeConfiguration.java

```diff
@@ -43,8 +44,10 @@ public class OutboundConnectorRuntimeConfiguration {
 
   @Bean
   public OutboundConnectorFactory outboundConnectorFactory() {
+    // filter out connectors that are disabled via additional env variable
     return new DefaultOutboundConnectorFactory(
-        OutboundConnectorDiscovery.loadConnectorConfigurations());
+        OutboundConnectorDiscovery.loadConnectorConfigurations(),
+        EnvVarsConnectorDisabler.getDisabledOutboundConnectorTypes());
```

Comment:
Curios: Wouldnt it be simpler for filter the list of configs before we handover the list to the factory so they are not even considered?

Reply:
I wanted to do this initially, but I think if a connector is disabled, I believe you shouldn't be able to add it as well. So the list of disabled connector to be preserved.